index.js

```diff
@@ -259,7 +263,7 @@ class Ora {
 		return fullPrefixText + frame + fullText;
 	}
 
-	clear() {
+	clear(stopping = false) {
```

Comment:
You should not expose this implementation detail to the user.

Reply:
Oh, I see what you mean, that's a public method. Hmm.